datafusion/src/logical_plan/builder.rs

```diff
@@ -696,21 +698,21 @@ impl LogicalPlanBuilder {
         let schema = schema.to_dfschema_ref()?;
 
         if analyze {
-            Ok(Self::from(LogicalPlan::Analyze {
+            Ok(Self::from(LogicalPlan::Analyze(AnalyzePlan {
```

Comment:
I think this code style is better than above style.
This is same with this pull request #1290